mozregression/telemetry.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import sys
+from multiprocessing import Process
+
+from mozlog import get_proxy_logger
+
+
+LOG = get_proxy_logger('telemetry')
+
+
+def _send_telemetry_ping(variant, appname):
+    from pathlib import Path
+    from glean import Glean, Configuration
+    from glean import (load_metrics,
+                       load_pings)
+    from mozregression import __version__
+
+    mozregression_path = Path.home() / '.mozilla' / 'mozregression'
+    Glean.initialize(
+        application_id="mozregression",
+        application_version=__version__,
+        upload_enabled=True,
+        configuration=Configuration(
+            ping_tag="mozregression-test-tag"
+        ),
+        data_dir=mozregression_path / "data"
+    )
+
+    # if we got this far, we can assume telemetry is enabled
+    Glean.set_upload_enabled(True)
+
+    pings = load_pings("pings.yaml")
+    metrics = load_metrics("metrics.yaml")
```

Comment:
Since you might send multiple pings per run, you might as well do this parsing at startup and save it in a global variable rather than parsing every time.

Also, you probably want to install these files as package data, and then retrieve them at runtime.  I think this will fail if running `mozregression` from a current working directory that doesn't contain these files.  See [including package data](https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files).  There's other ways to do that in Python if you don't want to use setuptools -- my point is just that they should be installed alongside the code so you don't have to run from a particular directory at run time.